### ISSUE TITLE
Performance on seperate test set over iterations

### ISSUE THREAD

**Issue by thomasvanmanen** [automl/auto-sklearn#471]
Hello,

Currently I am trying to measure the performance of Auto SKlearn over time on a seperate test set, to see if improvements in cross-validation error also lead to improvements when scored on a test set that the algorithm has not seen yet. To do this I would want to refit all the models that have been found in the SMBO, and then score these on this on a large simulated test set, to approximate the "true" test error. 

My question is then, how can I take an arbitrary iteration of the SMBO algorithm, train this configuration, and score it in a test set. Is it possible to generate an SKlearn pipeline similar to what is possible in TPOT, or is there another possibility?

Kind regards,
Thomas

**Comment by mfeurer** [comment]
Hi, unfortunately, this is not easily possible at the moment. We'll be working on this as port of https://github.com/automl/auto-sklearn/issues/426 and https://github.com/automl/auto-sklearn/issues/424.

Could you please give an example of what's possible in TPOT?

**Comment by thomasvanmanen** [comment]
In TPOT it is possible to do the following:


```

pipeline_str = list(tpot.evaluated_individuals_.keys())[i] # retrieve information of arbitrary trained pipeline,

# convert pipeline string to scikit-learn pipeline object
   
optimized_pipeline = creator.Individual.from_string(pipeline_str, tpot._pset) # obtain the correspoding object from the evolutionary algorithm
fitted_pipeline = tpot._toolbox.compile(expr=optimized_pipeline ) # convert evolutionary algorithm object to scikit-learn pipeline object

# Fit pipeline from scikit-learn

fitted_pipeline.fit(X_train, y_train) # Fit the pipeline like an ordinary scikit-learn pipeline
```

Having something similar in Auto SKlearn would be amazing!
